Lib/unittest/case.py

```diff
@@ -338,6 +339,19 @@ def __exit__(self, exc_type, exc_value, tb):
                 .format(logging.getLevelName(self.level), self.logger.name))
 
 
+def _short_description_from_docstring(text):
```

Comment:
You should use a shorter name like _make_short_description(doc)` or `_get_short_description(doc)`. It is unnecessarily verbose to include type information in the function name, e.g. as this can be obtained from the argument names and/or function docstring. Also, I note that `pydoc`'s convention is to use `doc` for the argument name.